Mods/SML/Source/SML/Public/Configuration/ConfigProperty.h

```diff
@@ -28,14 +29,24 @@ class SML_API UConfigProperty : public UObject {
     UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Configuration Property", meta = (MultiLine = true))
     FText Tooltip;
 
-	/** Whenever this value is only editable from main menu and disabled for editing in pause menu */
+	/** If this value is only editable from the main menu -> can't be edited from the pause menu when loaded into a game world */
 	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Configuration Property")
 	uint8 bRequiresWorldReload: 1;
 
-	/** Whenever this value should be hidden in Widgets ( No User Input )  */
+	/** If this value should not be displayed in Config Widgets, meaning the user can't see it to configure it (unless they manually edit the config file on disk)  */
 	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Configuration Property")
 	uint8 bHidden : 1;
 
+	/** If true, this property can be reset by the user from the Config Widget. */
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Configuration Property")
+	bool bAllowUserReset;
```

Comment:
This is set to true in all config property BPs, so it makes sense to just make it true here by default.

```suggestion
	bool bAllowUserReset = true;
```